OpenRA.Mods.Common/Traits/World/CreateMPPlayers.cs

```diff
@@ -63,13 +70,15 @@ void ICreatePlayersInfo.CreateServerPlayers(MapPreview map, Session lobbyInfo, L
 			}
 
 			// Create a player that is allied with everyone for shared observer shroud.
+			// We need to resolve the faction, even though we don't use it, to match the RNG state with clients
+			Player.ResolveFaction("Random", factions, playerRandom, false);
```

Comment:
Isn't `factions.First()` used as fallback instead of `"Random"` in the other places?

Reply:
https://github.com/OpenRA/OpenRA/blob/a375f0e58a0d82db4e752a4e700d8d8365304716/OpenRA.Mods.Common/Traits/World/CreateMPPlayers.cs#L110-L118

explicitly hardcodes "Random".

Reply:
I see.